sample-videochat-webrtc/src/main/res/layout/fragment_audio_conversation.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id = "@+id/fragmentIncomeCall"
+    style="@style/MatchParent"
+    >
+
+    <LinearLayout
+        android:id="@+id/info_about_call"
```

Comment:
https://github.com/QuickBlox/CodeStyle-Guidelines-for-Developers/blob/master/android/Resources.md#id-naming